leaderboard/src/test/java/xyz/breakit/leaderboard/grpc/LeaderboardGrpcServiceTest.java

```diff
@@ -0,0 +1,92 @@
+package xyz.breakit.leaderboard.grpc;
+
+import io.grpc.Server;
+import io.grpc.inprocess.InProcessChannelBuilder;
+import io.grpc.inprocess.InProcessServerBuilder;
+import io.grpc.testing.GrpcCleanupRule;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import xyz.breakit.leaderboard.*;
+import xyz.breakit.leaderboard.service.LeaderboardService;
+
+import java.io.IOException;
+import java.util.Iterator;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class LeaderboardGrpcServiceTest {
+
+    @Rule
+    public final GrpcCleanupRule grpcCleanup = new GrpcCleanupRule();
```

Comment:
Try using GrpcServerRule instead. Example: https://github.com/break-me-if-you-can/services/blob/master/geese/src/test/java/xyz/breakit/geese/GeeseServiceTest.java